runtime/doc/filetype.txt

```diff
@@ -512,30 +512,42 @@ Local mappings:
 
 MAN							*ft-man-plugin* *:Man*
 
-Displays a manual page in a nice way.  Also see the user manual
-|find-manpage|.
-
-To start using the ":Man" command before any manual page was loaded, source
-this script from your startup vimrc file: >
-
-	runtime ftplugin/man.vim
-
-Options:
-'iskeyword'	the '.' character is added to be able to use CTRL-] on the
-		manual page name.
+View manpages from the comfort of vim. Features include syntax highlighting,
+autocompletion, multilingual support, and manpage jumping.
+Also see |find-manpage|.
 
 Commands:
-Man {name}	Display the manual page for {name} in a window.
-Man {number} {name}
-		Display the manual page for {name} in a section {number}.
-
-Global mapping:
-<Leader>K	Displays the manual page for the word under the cursor.
+Man			Display the manual page for <cWORD> in a new tab.
+Man {name}		Display the manual page for {name} in a new tab.
+Man {sect} {name}	Same as above except specify the section.
+Man {name}({sect})	Alternate syntax to above.
 
 Local mappings:
-CTRL-]		Jump to the manual page for the word under the cursor.
-CTRL-T		Jump back to the previous manual page.
+CTRL-]			Jump to the manual page for the <cWORD> under the cursor.
+K			Same as CTRL-].
+CTRL-T			Jump back to the previous manual page.
+q			Close the window.
 
+Variables:
+g:man_find_window
+			If set, man will attempt to find the current man window
+			before opening a new one. A bang on the commands will
+			do the opposite of whatever this variable is set to.
+			Defaults to 1.
+g:man_synopsis
+			This only applies if you will be using languages other
+			than english.
+			In manpages of sections 2 and 3, there is sometimes some C
+			code under the heading SYNOPSIS. In english we can just match
+			SYNOPSIS and then can highlight all of the C code underneath.
+			However in other languages, SYNOPSIS is a different word, so how
+			are we to tell where the C code begins? This variable is a regex
+			that is used to match the SYNOPSIS heading.
+			Modify it for the language(s) you will be using.
+			Defaults to '\V\^SYNOPSIS\$'.
```

Comment:
This is unnecessary configuration and the amount of documentation required to describe it is disproportionate to its importance. It's trivial to extend the regex by adding extra words from other languages. If we want to support other languages by doing that, great. If new languages are to be supported, add more equivalents to the regex. But don't expose an option. The 5 people that actually take the time to discover this option could just as easily send a 1-line PR so that everyone viewing manpages in that language will benefit.